tests/functional/test_error_handling.py

```diff
@@ -15,9 +15,7 @@ def test_syntax_error_includes_module():
     with pytest.raises(exceptions.SourceSyntaxError) as excinfo:
         build_graph("syntaxerrorpackage", cache_dir=None)
 
-    expected_exception = exceptions.SourceSyntaxError(
-        filename=filename, lineno=5, text="fromb . import two\n"
-    )
+    expected_exception = exceptions.SourceSyntaxError(filename=filename, lineno=5, text="import")
```

Comment:
The `text` has changed here - the ruff parser reports a different text range for the syntax error than the python ast parser. See `let text = source_code.slice(e.location);` in the `import_parsing.rs` file.